components/IndoorDirections/TypesOfDirections/DifferentBuildingDirections.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { Line, G } from 'react-native-svg';
+import { ClassGraph } from '../../../constants/ClassGraph';
+
+export function DifferentBuildingDirections(props) {
+    const rooms = props.rooms;
+    const graph = ClassGraph();
+
+    const pathToElevator = dijkstra(graph, props.from, "elevator").path;
+    const pathToClass = dijkstra(graph, "elevator", "exit").path;
```

Comment:
You named the constant pathToClass, would it be possible to name it pathToExit since its third input is "exit"?

Reply:
Nice spot, I wanted to change it but never did!